doc/man1/list.pod

```diff
@@ -53,6 +54,13 @@ If a line is of the form
   foo => bar
 then B<foo> is an alias for the official algorithm name, B<bar>.
 
+=item B<-mac-algorithms>
+
+Display a list of message authentication code algorithms.
+If a line is of the form
+  foo => bar
```

Comment:
You will need an empty line before and after this one.  See with `perldoc doc/man1/list.pod` :wink:

Reply:
Huh?  It's coming out fine here:

```
    -digest-algorithms
        Display a list of message digest algorithms. If a line is of the form
        foo => bar then foo is an alias for the official algorithm name, bar.

    -mac-algorithms
        Display a list of message authentication code algorithms. If a line is
        of the form foo => bar then foo is an alias for the official algorithm
        name, bar.

    -cipher-commands
        Display a list of cipher commands, which are typically used as input
        to the dgst(1) or speed(1) commands.

    -cipher-algorithms
        Display a list of cipher algorithms. If a line is of the form foo =>
        bar then foo is an alias for the official algorithm name, bar.
```

Reply:
Or is this entire pod file formatted badly?

Reply:
Ah...  I thought the way to wrote it in source was an indication that you wanted `foo => bar` on a line of its own in output as well.